app/include/zmk/keys.h

```diff
@@ -20,6 +22,21 @@ struct zmk_key_event {
     bool pressed;
 };
 
+struct zmk_key_decoded {
```

Comment:
Not sure I love the decoded terminology... Complete key? Key details? Full key?

Reply:
Since this is all the data from a DT parameter, how about zmk_key_param?